package.json

```diff
@@ -31,6 +31,8 @@
   },
   "dependencies": {
     "is-is-odd": "^1.0.2",
-    "is-odd": "^3.0.1"
+    "is-odd": "^3.0.1",
+    "jquery": "^3.4.1",
+    "jquery-basic-arithmetic-plugin": "^1.1.0"
   }
```

Comment:
don't forget to add your name to the contributors field, so you are forever associated with this project.